reference/orders.v2.oas2.yml

```diff
@@ -3373,6 +3375,11 @@ components:
           description: The unique numeric identifier of the tax rate.
```

Comment:
🍹 probably we should mention about using `sales_tax_id` instead of this `tax_rate_id` field

```suggestion
          description: The unique numeric identifier of the tax rate. Note: this field has been deprecated, please use sales_tax_id instead
```

Reply:
Good idea @donald-nguyen-bc, thanks!